docs/articles/documentation/test-api/actions/right-click.md

```diff
@@ -21,14 +21,15 @@ The following example shows how to use the `t.rightClick` action to invoke a gri
 
 ```js
 import { expect } from 'chai';
+import { Selector } from 'testcafe';
 
 fixture `My fixture`
     .page `http://www.example.com/`;
 
 test('Popup Menu', async t => {
     await t.rightClick('#cell-1-1');
 
-    expect(await t.select('#cell-popup-menu')).to.be.null;
+    expect(await Selector('#cell-popup-menu')).to.be.null;
```

Comment:
This doesn't work. It should be `expect(await Selector('#cell-popup-menu')()).to.be.null;`

But I think our new built-in assertions will be awesome here:
```js
await t
    .rightClick('#cell-1-1')
    .expect(Selector('#cell-popup-menu').exists).eql(false);
```

Let's use it